apps/judicial-system/web/src/components/DefenderInfo/DefenderInfo.strings.ts

```diff
@@ -0,0 +1,96 @@
+import { defineMessages } from 'react-intl'
+
+export const defenderInfo = {
+  investigationCases: {
+    sections: {
+      defender: defineMessages({
+        heading: {
+          id:
+            'judicial.system.core:defender_info.investigation_cases.sections.defender.heading',
+          defaultMessage: 'Verjandi varnaraðila',
+          description:
+            'Notaður sem titill fyrir "upplýsingar um verjanda varnaraðila" hlutann á varnaraðila skrefi í rannsóknarheimildum.',
+        },
+        title: {
+          id:
+            'judicial.system.core:defender_info.investigation_cases.sections.defender.title',
+          defaultMessage: '{defenderType}',
+          description:
+            'Notaður sem titill fyrir "Verjandi/talsmaður" hlutann á fyrirtöku skrefi í rannsóknarheimildum.',
+        },
+        tooltip: {
+          id:
+            'judicial.system.core:defender_info.investigation_cases.sections.defender.tooltip',
+          defaultMessage:
+            'Lögmaður sem er valinn hér verður skipaður {defenderType} í þinghaldi og fær sendan úrskurð rafrænt.',
```

Comment:
Consider moving the conditional string to the contentful string. 
```suggestion
            'Lögmaður sem er valinn hér verður skipaður {sessionArrangement, select, ALL_PRESENT_SPOKESPERSON {talsmaður} other {verjandi}} í þinghaldi og fær sendan úrskurð rafrænt.',
```